pig/src/org/partiql/pig/domain/parser/TypeDomainParser.kt

```diff
@@ -15,275 +15,351 @@
 
 package org.partiql.pig.domain.parser
 
```

Comment:
For this file, I strongly recommend clicking the gear icon near the top and checking the box "hide whitespace changes" as most of the modified lines are just indentation.